test/tests/pbs_smoketest.py

```diff
@@ -83,7 +83,7 @@ def test_advance_reservation(self):
         r = Reservation(TEST_USER)
         now = int(time.time())
         a = {'Resource_List.select': '1:ncpus=4',
-             'reserve_start': now + 10,
+             'reserve_start': now + 20,
```

Comment:
I suggest using 30 instead of 20.  Some of the testbed machines are really slow.

Reply:
I agree, That's why I wanted to see failure logs first.

Reply:
Also, add an offset of 20 seconds while checking reservation in RUNNING state at line number 106.